indico/modules/rb/client/js/common/bookings/BookingDetails.jsx

```diff
@@ -912,6 +909,7 @@ class BookingDetails extends React.Component {
                   occurrenceCount={occurrenceCount}
                   recurrenceWeekdays={recurrenceWeekdays}
                 />
+                {renderPluginComponents('rb-booking-detail-extra-fields', {booking})}
```

Comment:
@VojtechPetru I changed this to pass the whole booking. Even if you just need the ID it seems better to expose everything just in ac se.

Reply:
ah, awesome, thank you! :)